lib/dailycrap.rb

```diff
@@ -33,7 +33,8 @@ def daily
         case event.type
         when 'PushEvent' 
           if event.payload.ref != 'refs/heads/master'
-            worked_on_prs << pr_from_ref(event.payload.ref) || event.payload.ref.gsub('refs/head/', '')
```

Comment:
There were 3 bugs.

- `refs/head` -> `refs/heads`
- change string to hash to have `title`
- `worked_on_prs << pr_from_ref(event.payload.ref)` is evalulated first